go/parquet/file/level_conversion.go

```diff
@@ -70,6 +70,7 @@ type LevelInfo struct {
 	RepeatedAncestorDefLevel int16
 }
 
+//lint:ignore U1000 newDefaultLevelInfo
 func newDefaultLevelInfo() *LevelInfo {
 	return &LevelInfo{NullSlotUsage: 1}
 }
```

Comment:
if this is unused, just delete it. worst case i can put it back in a future change if it's needed.